README.md

```diff
@@ -160,16 +208,16 @@ The Configurations Table will be used in conjunction with logging data in [Mothe
 
```

Comment:
We should add some additional parameters to the /sensors/logging/ and the /sensors/logging/:sensorId routes
- start time and end time so we can get logs from a certain time period
- limit so we can set a max number of logs to return
This will make it so we don't need the /recent route